app/views/devise/sessions/new.html.erb

```diff
@@ -5,11 +5,11 @@
 <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
   <fieldset>
     <div class="form-group">
-      <%= f.text_field :login, class: 'form-control', placeholder: t('.login'), autofocus: true %>
+      <%= f.text_field :login, class: 'form-control', placeholder: "Login", autofocus: true %>
```

Comment:
Hard-coding "Login" does not conform to our i18n guidelines. Our blogging platform needs to support an international audience of non-english language speakers.